src/commands/notify.yml

```diff
@@ -24,18 +24,78 @@ parameters:
     description: Notification message
     type: string
   priority:
-    description: 'Notification priority (low-to-high: -2 to 1)'
+    description: "Notification priority (low-to-high: -2 to 1)"
     type: integer
     default: 0
+  fail_only:
+    description: If `true`, notifications successful jobs will not be sent
+    type: boolean
+    default: false
+  only_for_branches:
+    description: If set, a comma-separated list of branches for which to send notifications. No spaces.
+    type: string
+    default: ""
 
 steps:
+  - run:
+      name: Pushover - Setting Failure Condition
+      command: |
+        echo 'export PUSHOVER_BUILD_STATUS="fail"' >> $BASH_ENV
+      when: on_fail
+
+  - run:
+      name: Pushover - Setting Success Condition
+      command: |
+        echo 'export PUSHOVER_BUILD_STATUS="success"' >> $BASH_ENV
+      when: on_success
```

Comment:
Is there a reason we can't ask users who want this behavior to do it themselves?

```yaml
      - pushover/notify:
          name: notify
          requires: [...]
          title: '...'
          message: 'Build failed'
          when: on_failure
```

We may need to accept our own `when` parameter and say `when: <<parameters.when>>`?

If none of that works, I wonder if making a new `pushover/notify-on-failure` command for this use-case would be a simpler option.

Reply:
I want to be notified only when the following `run(s)` have failed by line 87.
https://github.com/masutaka/circleci-bundle-update-pr/blob/fd3401ec320a6b72de9a83a11f69d6d499318a5c/.circleci/config.yml#L73-L88